chats/apps/sectorqueue/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class SectorqueueConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
+    name = 'chats.apps.sectorqueue'
```

Comment:
Change the app name to queues, doesn't need to reference the sectors.

Reply:
Name changed in apps folder and in api_v1 folder.